src/build.py

```diff
@@ -904,9 +912,11 @@ def OCaml():
 
 def Spec():
   buildbot.Step('spec')
-  # Spec builds in-tree. Always clobber and run the tests.
+  # Spec builds in-tree. Always clobber.
   proc.check_call(['make', 'clean'], cwd=ML_DIR)
-  proc.check_call(['make', 'all'], cwd=ML_DIR)
+  proc.check_call(['make', 'opt'], cwd=ML_DIR)
```

Comment:
Does `make opt` also build the spec interpreter? Afaik the waterfall uses that too.